_templates/layout.html

```diff
@@ -0,0 +1,41 @@
+{%- extends "alabaster/layout.html" %}
```

Comment:
What version of Sphinx do you use? Alabaster is included as a dependency in 1.3, and it looke like the the bug you're describing https://github.com/sphinx-doc/sphinx/issues/1794 was fixed in 1.3.2 - http://sphinx-doc.org/changes.html#release-1-3-2-released-nov-29-2015